test/modules/aws-account-integration.ts

```diff
@@ -41,9 +41,17 @@ describe('AwsAccount Integration Testing', () => {
     VALUES ('fake', 'creds', 'us-west-2')
   `));
 
-  it('does absolutely nothing when you apply this', apply);
+  // it('does absolutely nothing when you apply this', apply);
+
+  it('does absolutely nothing when you apply this', query(`
+    CALL iasql_schedule_apply();
+  `));
```

Comment:
Should we be testing `CALL iasql_schedule_sync();` too?

Reply:
yeah, I can call it even if it is a noop